examples/Learn_hyperparameters/nd_to_md_regression_direct_withcov.jl

```diff
@@ -476,20 +455,18 @@ end
     params_init = transform_unconstrained_to_constrained(priors, initial_params)#[1, :]
     println("Prior gives parameters between: [$(minimum(params_init)),$(maximum(params_init))]")
 
-    if isa(lambda, Real)
-        min_complexity = n_features_opt * log(lambda)
-        data = vcat(reshape(y[:, (n_train + 1):end], :, 1), 0.0, min_complexity) #flatten data
-    elseif isa(lambda, UniformScaling)
-        min_complexity = n_features_opt * log(lambda.λ)
-        data = vcat(reshape(y[:, (n_train + 1):end], :, 1), 0.0, min_complexity) #flatten data
-    else
-        min_complexity = (n_features_opt / output_dim) * 2 * sum(log.(diag(cholesky(lambda).L)))
-        # TODO: SOLVE EVAL PROBLEM TO FIT WITH THIS DATA. the following is just a fudge to get reasonable scaling and not a global truth
-        min_complexity *= 1
+    #=
+    min_complexity =
+        isa(lambda, UniformScaling) ? n_features_opt * log(lambda.λ) :
+        n_features_opt / output_dim * 2 * sum(log.(diag(cholesky(lambda).L)))
+    min_complexity = sqrt(abs(min_complexity))
 
-        data = vcat(reshape(y[:, (n_train + 1):end], :, 1), 0.0, min_complexity) #flatten data
-    end
+
+    data = vcat(reshape(y[:, (n_train + 1):end], :, 1), 0.0, min_complexity) #flatten data
     println("min_complexity: ", min_complexity)
+    =#
+    data = zeros(size(Γ, 1))
```

Comment:
Or maybe you are overwriting line 465 with line 468 here? In that case it seems correct as long as the forward map is defined by  `y[:, (n_train +1):end] - posterior_mean` in the first Np rows of the map.